core/block.js

```diff
@@ -1388,7 +1388,8 @@ Blockly.Block.newFieldImageFromJson_ = function(options) {
   var height =
     Number(Blockly.utils.replaceMessageReferences(options['height']));
   var alt = Blockly.utils.replaceMessageReferences(options['alt']);
-  return new Blockly.FieldImage(src, width, height, alt);
+  var flip_rtl = Blockly.utils.replaceMessageReferences(options['flip_rtl']);
```

Comment:
flip_rtl should always be a boolean, not a message reference, so this should be more like [this](https://github.com/google/blockly/blob/master/core/block.js#L1227) code:

```
  if (typeof options['spellcheck'] == 'boolean') {
    field.setSpellcheck(options['spellcheck']);
  }
```